### PR TITLE
Managed identity keyvault connection

### PR DESCRIPTION
**Pull request by ArtiomMatiom**

### CODE REVIEW

File: modules/azure/api_connectors/key_vault_managed_identity/main.tf [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  backend "azurerm" {}
+}
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group_template_deployment" "service_bus_managed_identity" {
```
Comment (JeffreyVWT) [inline modules/azure/api_connectors/key_vault_managed_identity/main.tf:18]: servicebus!

**Review by JeffreyVWT — CHANGES_REQUESTED** [review]

File: modules/azure/api_connectors/key_vault_managed_identity/connection.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    {
+      "type": "Microsoft.Web/connections",
+      "apiVersion": "2016-06-01",
+      "name": "[parameters('keyvault_connection_name')]",
+      "location": "[parameters('location')]",
+      "kind": "V1",
+      "properties": {
+          "displayName": "[parameters('keyvault_connection_name')]",
+          "parameterValueType": "Alternative",
+          "alternativeParameterValues": {
```
Comment (JeffreyVWT) [inline modules/azure/api_connectors/key_vault_managed_identity/connection.json:42]: Do you need this too ?

"parameterValueSet": {
          "name": "managedIdentityAuth",

**Review by JeffreyVWT — CHANGES_REQUESTED** [review]

File: modules/azure/api_connectors/key_vault_managed_identity/connection.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "keyvault_connection_name": {
+      "type": "string",
+      "metadata": {
+        "description": "Name to use for this connection"
+      }
+    },
+    "key_vault_name": {
+      "type": "string",
+      "metadata": {
+        "description": "Service Bus namespace"
```
[the remark this answers is not in the document]
Comment (ArtiomMatiom) [reply modules/azure/api_connectors/key_vault_managed_identity/connection.json:28]: changed to Keyvault Name for the connection target

File: modules/azure/api_connectors/key_vault_managed_identity/main.tf (hunk 1 above)
Comment (ArtiomMatiom) [reply modules/azure/api_connectors/key_vault_managed_identity/main.tf:18]: renamed to key_vault_managed_identity

File: modules/azure/api_connectors/key_vault_managed_identity/connection.json (hunk 2 above)
Comment (JeffreyVWT) [reply modules/azure/api_connectors/key_vault_managed_identity/connection.json:42]: Ok :)

File: modules/azure/api_connectors/key_vault_managed_identity/connection.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (JeffreyVWT) [reply modules/azure/api_connectors/key_vault_managed_identity/connection.json]: I still see
```"description": "Service Bus namespace"```

**Review by JeffreyVWT — CHANGES_REQUESTED** [review]

**Review by JeffreyVWT — APPROVED** [review]

**Pull request merged**